guide/additional-info/changes-in-v12.md

```diff
@@ -179,7 +179,7 @@ The method to ban members and users have been moved to the `GuildMemberManager`.
 
 ### Image URLs
 
-Some image-related properties like `user.avatarURL` are now a method in v12, so that you can apply some options to them, eg. to affect their display size. 
+Some image-related properties like `user.avatarURL` are now a method in v12 so that you can apply some options to them, e.g., to affect their display size. 
```

Comment:
```suggestion
Some image-related properties like `user.avatarURL` are now methods in v12 so that you can apply some options to them, e.g., to affect their display size. 
```